### YamlMime:UniversalReference
api_name: []
items:
- children:
  - azure.mgmt.datafactory.models.managed_integration_runtime_error.ManagedIntegrationRuntimeError
  fullName: azure.mgmt.datafactory.models.managed_integration_runtime_error
  langs:
  - python
  module: azure.mgmt.datafactory.models.managed_integration_runtime_error
  name: managed_integration_runtime_error
  source:
    id: managed_integration_runtime_error
    path: azure-mgmt-datafactory\azure\mgmt\datafactory\models\managed_integration_runtime_error.py
    remote:
      branch: master
      path: azure-mgmt-datafactory\azure\mgmt\datafactory\models\managed_integration_runtime_error.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 0
  type: module
  uid: azure.mgmt.datafactory.models.managed_integration_runtime_error
references:
- fullName: azure.mgmt.datafactory.models.managed_integration_runtime_error.ManagedIntegrationRuntimeError
  isExternal: false
  name: ManagedIntegrationRuntimeError
  parent: azure.mgmt.datafactory.models.managed_integration_runtime_error
  uid: azure.mgmt.datafactory.models.managed_integration_runtime_error.ManagedIntegrationRuntimeError
